app/controllers/staff/speakers_controller.rb

```diff
@@ -1,36 +1,26 @@
 class Staff::SpeakersController < Staff::ApplicationController
   decorates_assigned :speaker
-  before_filter :set_proposal, only: [:new, :create, :destroy]
+  before_filter :set_program_session, only: [:new, :create, :destroy]
```

Comment:
Do you need this on :destroy?

before_filter is phasing out so you'll want to switch them to before_action when updating existing controllers.